### PR TITLE
added mms2 directory with updated docker

### PR DESCRIPTION
**Pull request by guy9**

### CODE REVIEW

**Comment by guy9** [comment]
@tzach please review

File: mms2/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+## Scylla in Docker for the Mutant Monitoring Blog Series
```
Comment (tzach) [inline mms2/README.md]: Blog Series --> Scylla University

File: mms2/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+docker-compose kill
+docker-compose rm -f
+```
+### Importing the MMS keyspaces and data automatically 
+
+```
+docker exec scylla-node1 cqlsh -f /mutant-data.txt
+```
+
+Then re-run docker-compose:
```
Comment (tzach) [inline mms2/README.md]: Why do you need to re-run docker to load data?

Comment (guy9) [reply mms2/README.md]: I updated the readme file accordingly

File: mms2/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+### Instructions for setting up a Scylla Cluster from this repo
+
+```
+cd mms2
+docker-compose up -d
+```
+
+Run bash in the node:
+```
+docker exec -it mms_scylla-node1_1 bash
```
Comment (tzach) [inline mms2/README.md]: should it be mms2?
This does not work

File: mms2/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+### Destroying the Scylla Cluster 
+```
+cd mms2
+docker-compose kill
+docker-compose rm -f
+```
+### Importing the MMS keyspaces and data automatically 
+
+```
+docker exec mms_scylla-node1_1 cqlsh -f /mutant-data.txt
```
Comment (tzach) [inline mms2/README.md]: same here

Comment (guy9) [reply mms2/README.md]: You're right, fixed (I hope)

Comment (tzach) [reply mms2/README.md]: Please test

Comment (guy9) [reply mms2/README.md]: done, just retested it.

**Pull request merged**